tests/variable/variable.cpp

```diff
@@ -134,6 +151,34 @@ TEST(Variable, addTwoVariables) {
     EXPECT_EQ(e, a);
 }
 
+TEST(Variable, addVariables) {
+	const std::string e =
+		"<component>"
+		"<variable name=\"var1\" units=\"dimensionless\"/>"
+		"<variable name=\"var2\"/>"
+		"<variable name=\"var3\"/>"
```

Comment:
It might be worth having one of these with only units, or only initial_value, and no name, so more combinations are checked?